packages/amino/src/encoding.ts

```diff
@@ -45,13 +61,15 @@ export function encodeEd25519Pubkey(pubkey: Uint8Array): Ed25519Pubkey {
 // Prefixes listed here: https://github.com/tendermint/tendermint/blob/d419fffe18531317c28c29a292ad7d253f6cafdf/docs/spec/blockchain/encoding.md#public-key-cryptography
 // Last bytes is varint-encoded length prefix
 const pubkeyAminoPrefixSecp256k1 = fromHex("eb5ae987" + "21" /* fixed length */);
+const pubkeyAminoPrefixEthSecp256k1 = fromHex("eb5ae987" + "21" /* fixed length */);
```

Comment:
Did you test this prefix? Is this the one used in address derivation?

This is wrong in two ways:
1. eb5ae987 refers to tendermint/PubKeySecp256k1
2. 0x21 means that data of length 33 bytes is following

Reply:
EthSecp256 use the same prefix and length (please see https://github.com/cosmos/cosmjs/pull/1160#issuecomment-1264779586). 
Works and is available for trial via https://ezstaking.tools/.

Reply:
Thank you, good to know. Then this is broken in Ethermint. Possibly we could work around that by removing the reverse operation bytes -> Pubkey. Let's see ...